kinesis_video_streamer/src/streamer.cpp

```diff
@@ -23,6 +23,10 @@
 #include <kinesis_video_streamer/subscriber_callbacks.h>
 #include <ros/ros.h>
 
+using namespace Aws::Client;
+using namespace Aws::Kinesis;
```

Comment:
Don't utilize "using namespace" and, instead use either using or typedef to alias a type.

Reply:
The top priority was to remove `using namespace` from header files that we expose to other packages. At some point, let's remove `using namespace` from everywhere.